services/order.py

```diff
@@ -0,0 +1,43 @@
+import datetime
+
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.db.models import QuerySet
+from django.shortcuts import get_object_or_404
+
+from db.models import Order, Ticket
+
+
+@transaction.atomic
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: datetime = None
```

Comment:
The `date` parameter in `create_order` should have a default value of `None` to avoid potential issues if no date is provided. Consider changing the parameter to `date: datetime = None`.